diffcp/cone_program.py

```diff
@@ -1,4 +1,7 @@
 import warnings
+import os
+
+os.environ["OMP_THREAD_LIMIT"] = "1"
```

Comment:
Hmm, doing this globally could really mess with other libraries that need to use openmp, what happens if you set it back to whatever it was before after importing SCS here?

Reply:
That unfortunately doesn't work.

Reply:
`mkl.set_num_threads(1)` seems to work. (https://pypi.org/project/mkl/)